packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -19,39 +19,42 @@ export default class ModalDialog extends React.Component<Props> {
         const {style, children} = this.props;
         return (
             <MediaLayout spec={MEDIA_MODAL_SPEC}>
```

Comment:
We may actually want to move this up closer to the portal since eventually we'll want to render left/right chevrons on the modal backdrop itself and those may want access to the current media spec.